src/utils/storybook/fetchProductHandles.ts

```diff
@@ -0,0 +1,33 @@
+import { getText } from "@/utils/fetch"
+import { createShopifyUrl } from "@/utils/createShopifyUrl"
+
+export async function fetchProductHandles(rootUrl?: string) {
+  if (rootUrl) {
```

Comment:
drop the if branch, the Shopify object should be populated before this function is called